### PR TITLE
docs: elaborate on our dependency update policy

### PR DESCRIPTION
**Pull request by beyang**

### CODE REVIEW

File: docs/security.md [hunk 1]
```diff
@@ -66,6 +66,23 @@ We use a number of static analysis tools to identify security risks in developme
...
+- We monitor the Alpine Linux release notes to be aware of patches for CVEs and upgrade our base
+  Docker images as necessary.
+- If we are alerted to a vulnerability that exists in a version of Sourcegraph that would compromise
+  the privacy of a customer's code, we will notify the affected customers immediately. Patch
+  releases that fix the vulnerability will be prioritized as a P0, which means work will commence
+  immediately rather than waiting until the next planning phase.
+
+Customers are encouraged to update to the latest release of Sourcegraph every month. This will
+ensure that their Sourcegraph instance has the latest security-related updates. New releases are
+published on the 20th of every month, so customers that wish to stay up-to-date can set monthly
```
Comment (dadlerj) [inline docs/security.md:83]:
```suggestion
(published on the 20th of every month)[https://docs.sourcegraph.com/admin/updates], so customers that wish to stay up-to-date can set monthly
```

File: docs/security.md [hunk 2]
```diff
@@ -66,6 +66,23 @@ We use a number of static analysis tools to identify security risks in developme
...
+calendar reminders, or subscribe to the Sourcegraph newsletter.
```
Comment (dadlerj) [inline docs/security.md:84]:
```suggestion
calendar reminders, or (subscribe to the Sourcegraph newsletter)[https://about.sourcegraph.com/].
```

File: docs/security.md [hunk 3]
```diff
@@ -66,6 +66,23 @@ We use a number of static analysis tools to identify security risks in developme
 
 All development laptops have encrypted hard drives.
 
+## Updates
+
+Sourcegraph follows commonly recognized best practices for updating software dependencies.
```
Comment (dadlerj) [inline docs/security.md:71]:
```suggestion
Sourcegraph follows commonly-recognized best practices for updating software dependencies.
```

Comment (beyang) [reply docs/security.md:71]: Adverbs ending in `ly` should not be connected to the adjective they modify with a hyphen: https://prowritingaid.com/grammar/1000138/How-to-use-hyphen-after-adverbs.

**Pull request merged**